mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -45,7 +45,7 @@ CLASS ahw net/minecraft/entity/Entity
 	FIELD ax BREATH Lqe;
 	FIELD ay CUSTOM_NAME Lqe;
 	FIELD az NAME_VISIBLE Lqe;
-	FIELD b maxEntityId Ljava/util/concurrent/atomic/AtomicInteger;
+	FIELD b MAX_ENTITY_ID Ljava/util/concurrent/atomic/AtomicInteger;
```

Comment:
Uh no, atomic integer is mutable.

Reply:
It's `static final`. A field like `INSTANCE` is always uppercase despite being mutable, so your argument that uppercase is immutable doesn't work.